src/NorthwindWebsite/Presentation/Views/Categories/FromApiJS.cshtml

```diff
@@ -0,0 +1,26 @@
+﻿<h1>Categories from API with Javascript</h1>
+
+<script src="https://ajax.aspnetcdn.com/ajax/jQuery/jquery-2.0.3.min.js"></script>
+<link rel="stylesheet" href="~/css/entities-from-api.css" />
+
+<div>
+    <ul id="categories" />
+</div>
+
+<script>
+    let $categoriesList = $('#categories');
+    let apiUrl = '../api/categories/';
+    let method = 'GET';
+
+    $(function(){
+        $.ajax({
+        type: method,
+        url: apiUrl,
+        success: function(categories) {
+           $.each(categories, function(i, category){
+                $categoriesList.append('<li class="list-item">' + category.categoryName + "</li>");
```

Comment:
I think string template may be more suitable here: `<li class="list-item">'${category.categoryName }</li>`